targets/xiwi

```diff
@@ -402,7 +403,13 @@ EOF
     /usr/bin/install -s dummy_drv.so "$DRIVERDIR/"
 ) # End compilation subshell
 
-if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+if [ "${DISTROAKA:-"$DISTRO"}" = 'openSUSE' ]; then
+    if [ $ARCH = "aarch64" ] || [ $ARCH = "x86_64"]; then
+        cp /usr/lib/xorg/modules/drivers/* /usr/lib64/xorg/modules/drivers/
```

Comment:
It's for the Xorg dummy driver we install, all the other drivers should be fine.